include/fastdds/dds/xtypes/utils.hpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2024 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*!
+ * @file utils.hpp
+ */
+
+#ifndef FASTDDS_DDS_XTYPES_UTILS_HPP
+#define FASTDDS_DDS_XTYPES_UTILS_HPP
+
+#include <iostream>
+#include <string>
+
+#include <fastdds/dds/core/ReturnCode.hpp>
+#include <fastdds/dds/xtypes/dynamic_types/DynamicData.hpp>
+#include <fastdds/dds/xtypes/dynamic_types/DynamicType.hpp>
+
+namespace eprosima {
+namespace fastdds {
+namespace dds {
+
+enum class DynamicDataJsonFormat
+{
+    OMG,
+    EPROSIMA,
+};
+
+/**
+ * @brief Serializes a @ref DynamicType into its IDL representation.
+ *
+ * @param [in] dynamic_type The @ref DynamicType to serialize.
+ * @param [in,out] output The IDL representation.
```

Comment:
Your comments are always enigmatic. I don't know what you're talking about.

Reply:
I meant that I don't find the sentence very accurate, sounds to me like "output === The IDL representation", instead of something like "\c std::ostream reference where the IDL representation is inserted/streamed."